### DISCUSSION TITLE
If you can see this, traffic is not passing through mitmproxy error

### DISCUSSION THREAD

**Discussion by MeikyuuTrader** [mitmproxy/mitmproxy#5305] [Q&A] [answered]
Running Linux with Linux brew and installed mitmproxy via linux brew, following the startup instructions to run mitmproxy in cli and then going to mitm.it, I don't see any option to install the certificate. Instead I'm getting `If you can see this, traffic is not passing through mitmproxy` message. Also tried by downloading the linux binaries and then running the mitmproxy. Get the same error though.

When navigating to http://localhost:8080/, I get a 502 gateway error, but the mitmproxy can see that GET request in terminal. 

I also have Linux Valet installed for web development, not sure if that'd mess anything up with mtimproxy though. 

Mitmproxy Version: 8.0.0
Platform: Linux Ubuntu 20.04.4 LTS

**Comment by MeikyuuTrader** [comment, marked as answer]
Didn't read instructions properly. Had to enable network proxy to localhost:8080 in Network Settings then everything worked.

**Comment by Drjacky** [comment]
Still see the error even when I use: `adb shell settings put global http_proxy 127.0.0.1:8080` 😔

**Reply by mhils** [reply]
@Drjacky: The easiest alternative is WireGuard mode then. :)

**Reply by Drjacky** [reply]
It has been solved. I had to use the machine IP address(like 192.168.0.150) and **not** (127.0.0.1 or 10.0.2.2).

**Comment by xbetdepositteamkhan-sudo** [comment]
If you can see this, traffic is not going through mitmproxy.
